models/DataObject/ClassDefinition/Data/InputQuantityValue.php

```diff
@@ -189,4 +189,16 @@ public function denormalize($value, $params = [])
 
         return null;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isEmpty($data)
```

Comment:
InputQuantityValue extends QuantityValue. So this method should not be necessary.

Reply:
done